go/auth/token_test.go

```diff
@@ -0,0 +1,128 @@
+// Copyright 2015 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+package auth
+
+import (
+	"fmt"
+	"testing"
+
+	libkb "github.com/keybase/client/go/libkb"
+	keybase1 "github.com/keybase/client/go/protocol"
+)
+
+const testMaxTokenExpireIn = 60
+
+func TestTokenParse(t *testing.T) {
+	uid := keybase1.UID("01")
```

Comment:
in my branch, I have test routines to make UID/Name/KID pairs that are more like what the API server expects.